### PR TITLE
Fix libvirt group detection and program calling

### PR DESCRIPTION
**Pull request by LDprg**
Fixes for #95. Solves wrong invert when detecting the libvirt group. And fixed a wrong argumentwhen callibg freerdp.

### CODE REVIEW

**Comment by LDprg** [comment]
@oskardotglobal I did only write this on the go, so I couldn't test it.

**Comment by jrevillard** [comment]
Hello,

Do you finally tacle xfreerdp v2 or v3 ?.... You cannot support both simply as the syntax completely changed. In this MR, a move to v3 was done: https://github.com/winapps-org/winapps/pull/87 ... and here you go back to v2....

Best,
Jerome

File: bin/winapps [hunk 1]
```diff
@@ -88,10 +88,10 @@ if [ "${1}" = "windows" ]; then
 	$FREERDP_COMMAND ${RDP_FLAGS} /d:"${RDP_DOMAIN}" /u:"${RDP_USER}" /p:"${RDP_PASS}" /scale:${RDP_SCALE} +dynamic-resolution +auto-reconnect +home-drive /wm-class:"Microsoft Windows" /v:${RDP_IP} 1>/dev/null 2>&1 &
 elif [ "${1}" = "check" ]; then
 	dprint "CHECK"
-	$FREERDP_COMMAND ${RDP_FLAGS} /d:"${RDP_DOMAIN}" /u:"${RDP_USER}" /p:"${RDP_PASS}" /scale:${RDP_SCALE} +auto-reconnect +home-drive -wallpaper +dynamic-resolution ${MULTI_FLAG} /app:program:"explorer.exe" /v:${RDP_IP}
```
Comment (oskardotglobal) [inline bin/winapps:91]: This shouldn't be changed

File: bin/winapps [hunk 2]
```diff
@@ -114,9 +114,9 @@ elif [ "${1}" != "install" ]; then
 		dprint "HOME:${HOME}"
 		FILE=$(echo "${2}" | sed 's|'"${HOME}"'|\\\\tsclient\\home|;s|/|\\|g;s|\\|\\\\|g')
 		dprint "FILE:${FILE}"
-		$FREERDP_COMMAND ${RDP_FLAGS} /d:"${RDP_DOMAIN}" /u:"${RDP_USER}" /p:"${RDP_PASS}" /scale:${RDP_SCALE} +auto-reconnect +clipboard +home-drive -wallpaper +dynamic-resolution ${MULTI_FLAG} /wm-class:"${FULL_NAME}" /app:program:"${WIN_EXECUTABLE}",icon:"${ICON}",name:"${FULL_NAME}",cmd:"\"$FILE\"" /v:"${RDP_IP}" 1>/dev/null 2>&1 &
```
Comment (oskardotglobal) [inline bin/winapps:117]: Same thing here

File: bin/winapps [hunk 3]
```diff
@@ -114,9 +114,9 @@ elif [ "${1}" != "install" ]; then
...
+		$FREERDP_COMMAND ${RDP_FLAGS} /d:"${RDP_DOMAIN}" /u:"${RDP_USER}" /p:"${RDP_PASS}" /scale:${RDP_SCALE} +auto-reconnect +clipboard +home-drive -wallpaper +dynamic-resolution ${MULTI_FLAG} /wm-class:"${FULL_NAME}" /app:"${WIN_EXECUTABLE}",icon:"${ICON}",name:"${FULL_NAME}",cmd:"\"$FILE\"" /v:"${RDP_IP}" 1>/dev/null 2>&1 &
 	else
-		$FREERDP_COMMAND ${RDP_FLAGS} /d:"${RDP_DOMAIN}" /u:"${RDP_USER}" /p:"${RDP_PASS}" /scale:${RDP_SCALE} +auto-reconnect +clipboard +home-drive -wallpaper +dynamic-resolution ${MULTI_FLAG} /wm-class:"${FULL_NAME}" /app:program:"${WIN_EXECUTABLE}",icon:"${ICON}",name:"${FULL_NAME}" /v:"${RDP_IP}" 1>/dev/null 2>&1 &
```
Comment (oskardotglobal) [inline bin/winapps:119]: and here

**Review by oskardotglobal — CHANGES_REQUESTED** [review]
Yeah, @jrevillard is right
That program: syntax is needed for freerdp3, do not remove it

**Comment by oskardotglobal** [comment]
> Hello,
> 
> Do you finally tacle xfreerdp v2 or v3 ?.... You cannot support both simply as the syntax completely changed. In this MR, a move to v3 was done: #87 ... and here you go back to v2....
> 
> Best, Jerome

Just a small issue, this is why we review PRs
This of course will not be changed, we will be staying with freerdp3

File: bin/winapps (hunk 1 above)
Comment (LDprg) [reply bin/winapps:91]: whops forgot abor v2 v3. Will revert

**Comment by LDprg** [comment]
We might want to use a flag to switch between v2 and v3. Or we set v3 as official requirement, which is properly documented.

**Comment by LDprg** [comment]
I included the libvirt fix in the refactoring pr #98. If it gets merged this is not needed

**Comment by oskardotglobal** [comment]
Then we can close this PR, right?

**Comment by LDprg** [comment]
This is taken over by #98

**Pull request closed without merging**